tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/document/documents.controller.js

```diff
@@ -380,9 +403,22 @@
                         displayName: vm.documentConfigContent.GridColumn10Header,
                         headerCellClass: 'gridclass',
                         cellClass: 'gridclass',
-                        width: '170',
-                        visible: false,
-                        position: value.position
+                        width: '170',                        
+                        position: value.position,
+                        visible: value.defaultVisibleInGrid
+                    });
+                }
+            }
+            if (key == "documentPracticeGroup") {

```

Comment:
Modified the grid columns visibility based on the settings from config.js to cater the needs of both PC and MC